### DISCUSSION TITLE
advantages and disadvantages using multiple batteries or rather a singler bigger one

### DISCUSSION THREAD

**Discussion by hansgeorgk** [Louisvdw/dbus-serialbattery#1040] [Q&A]
Hi,
I wondereing what is the best approach to increase capacity of my 24V system.
Should I rather increase capacity of my existing battery by adding more cells. This could mean going from 8 cells straight 
<img width="526" alt="Bildschirmfoto 2024-04-19 um 20 26 09" src="https://github.com/Louisvdw/dbus-serialbattery/assets/100286224/62501fbe-b804-4a56-86fd-27102e2f5806">

to mixed parallel straight cell setup like this
<img width="513" alt="Bildschirmfoto 2024-04-19 um 20 26 44" src="https://github.com/Louisvdw/dbus-serialbattery/assets/100286224/1b707fbc-7154-4d38-8c25-9491af3fb717">

or would it be better to add a second 8 cell battery, with a second BMS?
any ideas, thoughts?
Probably I am not the first one to ask this and you can refer me to an exisisting discussion.

cheers
HG

**Comment by Albarge** [comment]
It's down to you really, many people still make parallel cell batteries, but I wouldn't. To me it seems BMS's are now so much better, more reliable, and cheaper that there really is little point in not having individual cell data and protection with separate batteries especially if you are going to use serialbattery.

The [DIYSolarforum](https://diysolarforum.com/) is probably a good place to ask though.

**Comment by rolandlueders** [comment]
First i had two 2p4s Batteries 12V with the option to 24V as 8s Batteries. As i reassambled the batteries to 24V, i saw that they were not good balanced. Before as 12V Batteries they were good balanced. It seems to be better if every cell has an balancer. And a plus is the higher amperage.
